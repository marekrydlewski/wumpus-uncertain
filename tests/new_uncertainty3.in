4 4
0.1
?B??
?BB?
B?B?
B?B?
