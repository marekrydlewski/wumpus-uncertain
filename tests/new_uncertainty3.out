0.17 0.00 0.92 0.10 
0.17 0.00 0.00 0.17 
0.00 0.92 0.00 0.17 
0.00 1.00 0.00 0.10 
